remove qtauto image from available images / reformat README

tobsan opened the pull request:
With b2qt being conditional in meta-pelux, we only show this image as
suggested image when we are building with b2qt support. This requires a
change in the manifest to copy that file.

Signed-off-by: Tobias Olausson <tobias.olausson@pelagicore.com>

GordanM commented on README.md:
```diff
@@ -6,12 +6,21 @@ Maintained at https://github.com/pelagicore/meta-pelux
 
-License information for any other files is either explicitly stated or defaults to MIT license.
+License information for any other files is either explicitly stated or defaults
+to MIT license.
```
Formatting changes should be a separate commit.

Commit:
conf-notes: remove qtauto image from available images

With b2qt being conditional in meta-pelux, we only show this image as
suggested image when we are building with b2qt support. This requires a
change in the manifest to copy that file.

Signed-off-by: Tobias Olausson <tobias.olausson@pelagicore.com>

Commit:
README.md: wraps text

Signed-off-by: Tobias Olausson <tobias.olausson@pelagicore.com>

tobsan commented:
@GordanM separated the commits now.
